stage #2/module #4 String/lesson #1. String/content.md

```diff
@@ -0,0 +1,200 @@
+| int hashCode()                                                                             | Returns a hash code for this string.                                                                                                        |
+| int indexOf(int ch)                                                                        | Returns the index within this string of the first occurrence of the specified character.                                                    |
+| int indexOf(int ch, int fromIndex)                                                         | Returns the index within this string of the first occurrence of the specified character, starting the search at the specified index.        |
+| int indexOf(String str)                                                                    | Returns the index within this string of the first occurrence of the specified substring.                                                    |
+| int indexOf(String str, int fromIndex)                                                     | Returns the index within this string of the first occurrence of the specified substring, starting at the specified index.                   |
+| String intern()                                                                            | Returns a canonical representation for the string object.                                                                                   |
+| boolean isEmpty()                                                                          | Returns true if, and only if, length() is 0.                                                                                                |
+| static String join(CharSequence delimiter, CharSequence... elements)                       | Returns a new String composed of copies of the CharSequence elements joined together with a copy of the specified delimiter.                |
+| static String join(CharSequence delimiter, Iterable<? extends CharSequence> elements)      | Returns a new String composed of copies of the CharSequence elements joined together with a copy of the specified delimiter.                |
+| int lastIndexOf(int ch)                                                                    | Returns the index within this string of the last occurrence of the specified character.                                                     |
+| int lastIndexOf(int ch, int fromIndex)                                                     | Returns the index within this string of the last occurrence of the specified character, searching backward starting at the specified index. |
+| int lastIndexOf(String str)                                                                | Returns the index within this string of the last occurrence of the specified substring.                                                     |
```

Comment:
Probably it's better to put here only some main methods instead of all available. 
It's difficult to read such a big table

Reply:
adjusted, thanks